src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,89 @@
+package parser;
+
+import command.AddCommand;
+import command.Command;
+import command.DeleteCommand;
+import command.DoneCommand;
+import command.ExitCommand;
+import command.FindCommand;
+import command.ListCommand;
+
+import exceptions.DukeException;
+
+public class Parser {
```

Comment:
Similar to previous suggestion, although it may not be necessary, but perhaps can consider to include comment/brief description of `Parser` class?

Reply:
Yes I think it would be good to have a description too. Will add in future commits.